src/test/java/org/xstefank/model/PersistentListTest.java

```diff
@@ -30,8 +30,8 @@ public void before() {
 
     @Test
     public void testIfUserIsSaved() {
-        testUserList.addUser(USERNAME);
-        Assert.assertTrue(testUserList.hasUsername(USERNAME));
+        testUserList.add(USERNAME);
```

Comment:
maybe it would be worthwhile to rename these tests to not reference userlist anymore?